modules/manage/pages/cluster-maintenance/compaction-settings.adoc

```diff
@@ -26,6 +26,47 @@ Compaction policy may be applied to a cluster or to an individual topic. If both
 * `compact`: This triggers only cleanup of messages with multiple versions. A message that represents the only version for a given key is not deleted.
 * `compact,delete`: This combines both policies, deleting messages exceeding the retention period while compacting multiple versions of messages.
 
+== Tombstone record removal
+
+When all messages for a given key are deleted, the topic partition keeps one message, called a tombstone, that contains the key, and a null value. A tombstone record indicates to clients that the messages have been deleted. You can specify how long Redpanda retains these tombstones for compacted topics using both a cluster configuration property `tombstone_retention_ms` and a topic configuration property xref:reference:properties/topic-properties.adoc#deleteretentionms[`delete.retention.ms`]. If both are set, the topic-level tombstone retention policy overrides the cluster-level policy.
+
+[NOTE]
+====
+Redpanda does not remove tombstone records for compacted topics with Tiered Storage enabled. 
```

Comment:
Can we provide an explanation for why?

Reply:
I don't think we should- tiered storage tombstone removal is a very large undertaking/project that we don't explicitly have as a current focus.

Perhaps phrasing it as "Redpanda does not currently remove tombstone records for compacted topics with Tiered Storage enabled" would be best, cc: @mattschumpert 

This line may also not be needed since you mention the configuration properties cannot be enabled in the same note.

Reply:
No, there is currently a hard restriction on tombstone removal being enabled along with tiered storage properties on a topic. No tombstones are removed for any TS enabled topics.